LoopKitUI/Views/Information Screens/BasalRatesInformationView.swift

```diff
@@ -34,8 +34,10 @@ public struct BasalRatesInformationView: View {
         VStack(alignment: .leading, spacing: 25) {
             Text(LocalizedString("Your Basal Rate of insulin is the number of units per hour that you want to use to cover your background insulin needs.", comment: "Information about basal rates"))
             Text(String(format: LocalizedString("%1$@ supports 1 to 48 rates per day.", comment: "Information about max number of basal rates"), appName))
-            Text(LocalizedString("The schedule starts at midnight and cannot contain a rate of 0 U/hr.", comment: "Information about basal rate scheduling"))
-        }
+            Text(LocalizedString("The schedule starts at midnight and cannot contain a single rate of 0 U/hr.", comment: "Information about basal rate scheduling"))
```

Comment:
This is pump specific. Until we can adapt and provide pump specific advice, the lowest common denominator functionality should probably be described here.